openmdao/recorders/base_case_reader.py

```diff
@@ -45,11 +59,11 @@ def system_metadata(self):
         Returns
         -------
         dict
-            reference to the 'system_options' attribute.
+            reference to the '_system_options' attribute.
         """
         warn_deprecation("The BaseCaseReader.system_metadata attribute is deprecated. "
                          "Use the BaseCaseReader.system_option attribute instead.")
```

Comment:
this should say to use `list_model_options` instead of the _other_ deprecated attribute ;)